internal/pkg/errors/errors.go

```diff
@@ -138,8 +138,28 @@ The profile name can only contain lowercase letters, numbers, and "-" and cannot
 
 To enable it, run:
   $ stackit %s enable`
+
+	IAAS_SERVER_MISSING_VOLUME_SIZE = `Boot volume size must be provided when "source_type" is "image".`
+
+	IAAS_SERVER_MISSING_IMAGE_OR_VOLUME_FLAGS = `Either Image ID or boot volume flags must be provided.`
```

Comment:
We should be a bit more specific here so the user know exactly what they need to provide. Is it all the "boot-volume-..." flags that are needed? Or just some?

Reply:
E.g. if all are needed:

```suggestion
	IAAS_SERVER_MISSING_IMAGE_OR_VOLUME_FLAGS = `Either "image-id" or the "boot-volume-..." flags must be provided.`
```

Reply:
image-id or boot-volume-source-id and boot-volume-source-type must be provided. I adapted the message as : 

	IAAS_SERVER_MISSING_IMAGE_OR_VOLUME_FLAGS = `Either "image-id" or "boot-volume-source-type" and "boot-volume-source-id" flags must be provided.`